### PR TITLE
Adm-938 [frontend]: Add a mark to the failed time range in report page

### PR DESCRIPTION
**Pull request by Leiqiuhong**
## Summary

Adm-938 [frontend]: Add a mark to the failed time range in report page

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Review by weiraneve — APPROVED** [review]
LGTM

**Review by mrcuriosity-tw — APPROVED** [review]
LGTM.

File: frontend/__tests__/components/Common/DateRangeViewer/DateRangeViewer.test.tsx [hunk 1]
```diff
@@ -62,51 +66,103 @@ describe('DateRangeViewer', () => {
...
+      const { getByLabelText } = setup(mockDateRanges);
+      expect(screen.getByTestId('PriorityHighIcon')).toBeInTheDocument();
 
-    await userEvent.click(getByLabelText('expandMore'));
-    expect(screen.getAllByTestId('PriorityHighIcon')).toHaveLength(4);
+      await userEvent.click(getByLabelText('expandMore'));
+      expect(screen.getAllByTestId('PriorityHighIcon')).toHaveLength(4);
+    });
+
+    it('should not show priority high icon when click expand button', async () => {
```
Comment (guzhongren) [inline frontend/__tests__/components/Common/DateRangeViewer/DateRangeViewer.test.tsx:96]: 这个测试的描述和上一个测试的描述基本一致，还有什么条件？

File: frontend/__tests__/components/Common/DateRangeViewer/DateRangeViewer.test.tsx [hunk 2]
```diff
@@ -62,51 +66,103 @@ describe('DateRangeViewer', () => {
...
-    await userEvent.click(getByLabelText('expandMore'));
+      store.dispatch(updateReportPageFailedTimeRangeInfos(failedTimeRangeList));
+      const { getByLabelText } = setup(mockDateRanges);
+
+      await userEvent.click(getByLabelText('expandMore'));
+
+      expect(screen.queryByTestId('PriorityHighIcon')).not.toBeInTheDocument();
+    });
+
+    it('should show priority high icon in report page when click expand button', async () => {
```
Comment (guzhongren) [inline frontend/__tests__/components/Common/DateRangeViewer/DateRangeViewer.test.tsx:145]: given。zzzz

**Review by guzhongren — COMMENTED** [review]
comments

**Review by neomgb — APPROVED** [review]

**Pull request merged**